source/agora/test/Base.d

```diff
@@ -185,8 +187,15 @@ private UnitTestResult customModuleUnitTester ()
     foreach (mod; single_threaded)
         runTest(mod);
 
-    foreach (mod; parallel(parallel_tests))
-        runTest(mod);
+    version (Windows)
+    {
+        // TODO
+    }
+    else
+    {
+        foreach (mod; parallel(parallel_tests))
+            runTest(mod);
+    }
```

Comment:
No! The code I gave you yesterday would run the test single-threaded. This one just excludes the test, which is absolutely not what we want.

Reply:
Yes, I proceeded with the missing test. 
In the Windows version, modify to test only single threads.